pallets/parachain-staking/src/lib.rs

```diff
@@ -1199,29 +1238,27 @@ pub mod pallet {
 			Ok(().into())
 		}
 		#[pallet::weight(<T as Config>::WeightInfo::set_blocks_per_round())]
-		/// Set blocks per round
+		/// Update the round configuration.
 		/// - if called with `new` less than length of current round, will transition immediately
 		/// in the next block
 		/// - also updates per-round inflation config
-		pub fn set_blocks_per_round(origin: OriginFor<T>, new: u32) -> DispatchResultWithPostInfo {
+		pub fn set_blocks_per_round(origin: OriginFor<T>, new_round: RoundInfo<T::BlockNumber>) -> DispatchResultWithPostInfo {
```

Comment:
this was an aggressive change, would need to rename to
```suggestion
		pub fn put_round_info(origin: OriginFor<T>, new_round: RoundInfo<T::BlockNumber>) -> DispatchResultWithPostInfo {
```